src/entity/Feed.ts

```diff
@@ -35,6 +40,17 @@ export class Feed {
   @Index('IDX_feed_slug', { unique: true })
   slug: string;
 
+  @Column({
+    type: 'text',
+    update: false,
+    insert: false,
+    nullable: false,
+    unique: false,
+    generatedType: 'STORED',
+    asExpression: `CASE WHEN "id" = "userId" THEN 'main' ELSE 'custom' END`,
+  })
+  type: FeedType;
```

Comment:
hmm I thought to put it to graphorm, but I guess this works as well 👍

Reply:
Didn't think of using GraphORM, though think it's nicer to keep it here, so could use it for for SQL stuff.